pyomo/contrib/pynumero/sparse/__init__.py

```diff
@@ -0,0 +1,37 @@
+#  ___________________________________________________________________________
+#
```

Comment:
@jsiirola this copyright statement appears on some of the files, should it be included on every file here?